src/ansys/geometry/core/designer/geometry_commands.py

```diff
@@ -496,6 +498,28 @@ def extrude_edges_up_to(
 
     @protect_grpc
     @min_backend_version(25, 2, 0)
+    def rename_object(self, selection: List[EntityIdentifier], name: str) -> bool:
```

Comment:
Type hint is not correct. Also, we should be passing in the ``Body``, ``Component``, ``Face``, ``Edge`` objects rather than a list of strings. Internally (within this method) we should extract the IDs. It might imply creating a generic method which we can use elsewhere